config/zones/SE.yaml

```diff
@@ -90,6 +94,9 @@ emissionFactors:
       - datetime: '2022-01-01'
         source: EU-ETS, ENTSO-E 2022
         value: 925.95
+      - datetime: '2023-01-01'
+        source: EU-ETS, ENTSO-E 2023
+        value: 373.49
```

Comment:
This change is due to Karlshamnverket having lowered the emissions a lot from 171,331 in 2022 to 12,044 in 2023 (92% decrease).